lldb/source/Plugins/Language/CPlusPlus/LibCxx.cpp

```diff
@@ -456,9 +464,14 @@ ExtractLibcxxStringInfo(ValueObject &valobj) {
   if (!l)
     return {};
 
-  StringLayout layout = l->GetIndexOfChildWithName("__data_") == 0
-                            ? StringLayout::DSC
-                            : StringLayout::CSD;
+  auto index_or_err = l->GetIndexOfChildWithName("__data_");
+  if (!index_or_err) {
+    LLDB_LOG_ERROR(GetLog(LLDBLog::Types), index_or_err.takeError(), "{0}");
```

Comment:
I know other places in this component use the `Types` log but it would make more sense to use the `DataFormatters` log channel (i'll clean it up in other places to make them all log to `DataFormatters`)